src/web/mjs/connectors/Comikey.mjs

```diff
@@ -0,0 +1,90 @@
+import Connector from '../engine/Connector.mjs';
+
+export default class ComiKey extends Connector {
+
+    constructor() {
+        super();
+        super.id = 'comikey';
+        super.label = 'Comikey';
+        this.tags = ['manga', 'webtoon', 'english'];
+        this.url = 'https://comikey.com';
+        this.chapterUrl = 'https://relay-us.epub.rocks/consumer/COMIKEY/series/';
+    }
+
+    _getMangaListFromPages(page) {
+        page = page || 1;
+        let uri = new URL('/sapi/comics', this.url);
+        uri.searchParams.set('page', page);
+        let request = new Request(uri.href, this.requestOptions);
+        return this.fetchJSON(request)
+            .then(data => {
+                let mangaList = data.results.map(manga => {
+                    return {
+                        id: manga.id + ',' + manga.e4pid,
+                        title: manga.name
+                    };
+                });
+                if (mangaList.length == 25) {
+                    return this._getMangaListFromPages(page + 1)
+                        .then(mangas => mangaList.concat(mangas));
+                } else {
+                    return Promise.resolve(mangaList);
+                }
+            });
+    }
+
+    _getMangaList(callback) {
+        this._getMangaListFromPages()
```

Comment:
Avoid recursive calls in favor of a loop, see:
- https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/templates/%40Samples.mjs#L48-L55
- https://github.com/manga-download/hakuneko/blob/master/src/web/mjs/connectors/templates/%40Samples.mjs#L76-L86